student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, name = 'unknown')
+    super(age, parent_permission, name)
+    @classroom = classroom
+  end

```

Comment:
- [ ] You did not set `age` and `parent_permission` parameters in your `initialize` method. Kindly add them.

- [ ] To avoid unnecessary future errors, I would recommend you re-arrange the order of the `super()` method arguments to be uniform with the Person class `initialize` method.